### ISSUE TITLE
PS + O combo buttons doesn't work on 

### ISSUE THREAD

**Issue by schyhub** [ShadowBlip/OpenGamepadUI#260]
```
chimera-session
  desktop
  desktop-xorg
  gamepadui
  opengamepadui
* gamepadui-with-qam
```
```sudo evtest
sudo evtest
/dev/input/event13:     DualSense Wireless Controller
/dev/input/event14:     DualSense Wireless Controller Motion Sensors
/dev/input/event15:     DualSense Wireless Controller Touchpad
```
choose
13

evtest is filled with non-stop spamming of 
```
Event: time 1695031565.938056, type 3 (EV_ABS), code 1 (ABS_Y), value 127
Event: time 1695031565.938056, -------------- SYN_REPORT ------------
Event: time 1695031565.938059, type 3 (EV_ABS), code 1 (ABS_Y), value 126
Event: time 1695031565.938059, -------------- SYN_REPORT ------------
Event: time 1695031565.948057, type 3 (EV_ABS), code 0 (ABS_X), value 126
Event: time 1695031565.948057, -------------- SYN_REPORT ------------
Event: time 1695031565.948060, type 3 (EV_ABS), code 0 (ABS_X), value 127
Event: time 1695031565.948060, -------------- SYN_REPORT ------------
Event: time 1695031565.948068, type 3 (EV_ABS), code 1 (ABS_Y), value 127
Event: time 1695031565.948068, -------------- SYN_REPORT ------------
Event: time 1695031565.954720, type 3 (EV_ABS), code 0 (ABS_X), value 126
Event: time 1695031565.954720, type 3 (EV_ABS), code 1 (ABS_Y), value 126
Event: time 1695031565.954720, -------------- SYN_REPORT ------------
Event: time 1695031565.954723, type 3 (EV_ABS), code 0 (ABS_X), value 127
Event: time 1695031565.954723, type 3 (EV_ABS), code 1 (ABS_Y), value 127
Event: time 1695031565.954723, -------------- SYN_REPORT ------------
Event: time 1695031565.954727, type 3 (EV_ABS), code 1 (ABS_Y), value 126
Event: time 1695031565.954727, -------------- SYN_REPORT ------------
Event: time 1695031565.961389, type 3 (EV_ABS), code 1 (ABS_Y), value 127
Event: time 1695031565.961389, -------------- SYN_REPORT ------------
Event: time 1695031565.961391, type 3 (EV_ABS), code 1 (ABS_Y), value 126
Event: time 1695031565.961391, -------------- SYN_REPORT ------------
Event: time 1695031565.968056, type 3 (EV_ABS), code 1 (ABS_Y), value 127
```

when I get to pressed the combo, this is what I receive
```
Event: time 1695031565.968056, -------------- SYN_REPORT ------------
Event: time 1695031565.968068, type 1 (EV_KEY), code 316 (BTN_MODE), value 0
Event: time 1695031565.968068, -------------- SYN_REPORT ------------
Event: time 1695031565.968072, type 3 (EV_ABS), code 1 (ABS_Y), value 126
Event: time 1695031565.968072, -------------- SYN_REPORT ------------
Event: time 1695031565.968074, type 1 (EV_KEY), code 305 (BTN_EAST), value 0
```

Wonder  if the spamming is why there the combo buttons doesn't work

**Comment by pastaq** [comment]
that spam is just stick drift.

I see a clue as to what is going on. Only the "value 0" events are making it through so the logic is not working in our input manager somewhere. I should be getting one of these dualsense controllers soon to help troubleshoot this issue.

**Comment by pastaq** [comment]
I recently got a dualshock 4 and a dualsense controlelr to figure out what is happening here. Turns out the way we were detecting virtual controllers was slightly flawed as the driver for the PS controllers doesn't assign a phys address to the evdev. Knowing this we can check for virtual devices another way and solve this issue soon.

**Comment by schyhub** [comment]
Interesting. Then I hope it will be resolved soon.

Thank for the works!

**Comment by ShadowApex** [comment]
Fixed in #264
